pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -762,6 +763,13 @@ func (m *resourceAllocator) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle
 		if m.cpuManager != nil {
 			err = m.cpuManager.Allocate(pod, &container)
 			if err != nil {
+				if _, ok := err.(*topologymanager.SMTAlignmentError); ok {
+					return lifecycle.PodAdmitResult{
+						Message: err.Error(),
+						Reason:  "SMTAlignmentError",
+						Admit:   false,
+					}
+				}
```

Comment:
I'm fine with this as is, but I'm curious how pervasive the use of "error wrapping" is in the Kubernetes code base: https://blog.golang.org/go1.13-errors#TOC_3.3.

This would be a cleaner solution, but I don't want to introduce something not widely used in K8s yet.